src/main/java/spring/core/dao/UserDao.java

```diff
@@ -0,0 +1,10 @@
+package spring.core.dao;
+
+import java.util.List;
+import spring.core.model.User;
+
+public interface UserDao {
+    void add(User user);
+
+    List<User> listUsers();
```

Comment:
This method shoud be named as "getAll()"

Reply:
changed.